deepmd/pt/utils/env_mat_stat.py

```diff
@@ -142,9 +145,11 @@ def iter(
                     self.descriptor.get_ntypes(), device=env.DEVICE, dtype=torch.int32
                 ).view(-1, 1),
             )
+            # shape: (ntypes, nloc, nnei)
+            type_idx = (type_idx.unsqueeze(-1) * exclude_mask) == 1
```

Comment:
I don't understand here. It seems the exclude types will be 0? However, 0 is the first type, and it should be -1.

Reply:
Oh, I think I understand now.

Reply:
I think [`torch.logical_and`](https://pytorch.org/docs/stable/generated/torch.logical_and.html) is much clearer.